typescript/s3-sns-lambda-chain/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "s3_sns_sqs_lambda_chain",
+  "version": "0.1.0",
+  "bin": {
+    "s3_sns_sqs_lambda_chain": "bin/s3_sns_sqs_lambda_chain.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "cdk": "cdk"
+  },
+  "devDependencies": {
+    "@aws-cdk/assertions": "*",
+    "@types/jest": "^26.0.10",
+    "@types/node": "10.17.27",
+    "aws-cdk": "*",
+    "jest": "^26.4.2",
+    "ts-jest": "^26.2.0",
+    "ts-node": "^9.0.0",
+    "typescript": "~3.9.7"
+  },
+  "dependencies": {
+    "@aws-cdk/aws-lambda-nodejs": "*",
+    "@aws-cdk/aws-lambda-event-sources": "*",
+    "@aws-cdk/aws-s3": "*",
+    "@aws-cdk/aws-s3-notifications": "*",
+    "@aws-cdk/aws-sns": "*",
+    "@aws-cdk/aws-sns-subscriptions": "*",
+    "@aws-cdk/aws-sqs": "*",
+    "@aws-cdk/core": "*"
+  }
```

```suggestion
  "devDependencies": {
    "@types/jest": "^26.0.10",
    "@types/node": "10.17.27",
    "aws-cdk": "*",
    "jest": "^26.4.2",
    "ts-jest": "^26.2.0",
    "ts-node": "^9.0.0",
    "typescript": "~3.9.7"
  },
  "dependencies": {
   	"aws-cdk-lib": "*",
    "constructs": "*"
  }
```

Would you mind [upgrading this to CDK v2](https://docs.aws.amazon.com/cdk/v2/guide/migrating-v2.html#migrating-v2-v1-uppgrade) for consistency with the other projects we have? I've added code suggestions with all the necessary code to upgrade. It's possible I may have missed something though but let me know if you have trouble.

By the way you'll need to update the lockfile once these suggestions are accepted. Run `npm install` and commit the updated lockfile changes.